src/lib/logger.js

```diff
@@ -1,12 +1,16 @@
 import { readFileSync } from 'fs'
-import path from 'path'
+import path, { dirname } from 'path'
+import { fileURLToPath } from 'url'
 import util from 'util'
 import chalk from 'chalk'
 import emojis from 'emojis'
 import wordWrap from 'word-wrap'
 
 export { chalk }
 
+const __filename = fileURLToPath(import.meta.url)
+const __dirname = dirname(__filename)
```

Comment:
Why you use the `import.meta...` not classic `__dirname`?

Reply:
https://nodejs.org/api/esm.html#esm_no_filename_or_dirname

Reply:
This is one of the trade-offs of using ES Modules. I hope they come up with a solution soon :/

Reply:
LOL I always transform ESM to CJS in Node and I don't know about it. Good to know. Thanks!